tests/test_cases/test_iteration_verilog/test_iteration_es.py

```diff
@@ -38,6 +38,8 @@ def recursive_discovery(dut):
                                            "ncsim")):
         # vpiAlways does not show up in IUS
         pass_total = 259
+    elif cocotb.SIM_NAME.lower().startswith(("chronologic simulation vcs")):
```

Comment:
These extra parentheses are pointless

Reply:
This was indeed meant as a preparation (and a mental hint) for when there is more than a single string to test where you'd use a tuple, without actually creating a tuple yet.

Reply:
Since the other code in this file does it the same way I'm fine with it for now (even it doesn't look nice at all).